docs/operating-scylla/nodetool-commands/stop.rst

```diff
@@ -7,7 +7,7 @@ Usage
 
 .. code:: sh
 
-          nodetool <options> stop -- <compaction_type>
+          nodetool <options> stop <compaction_type> 
```

Comment:
could you add a prefix in the title of the commit message and the pull request, so they would look like:

> doc: use uppercase compaction_type in nodetool command line
>
> <more details here>
>
> Signed-off-by: ...